ai/Algorithm/evaluation_module.py

```diff
@@ -51,7 +50,7 @@ def is_ball_moving(min_speed=0.1):
 
 def is_ball_our_side():
     # Retourne TRUE si la balle est dans notre demi-terrain
-    if GameState().field.our_side == FieldSide.POSITIVE:
+    if GameState().field.our_side == 0: # POSITIVE
```

Comment:
Pourquoi ne pas utiliser l'enum? il faut faire: if `GameState().field.our_side is FieldSide.POSITIVE:`